Libs/Gst/Classes/Parse.cs

```diff
@@ -10,16 +10,24 @@ public static class Parse
         /// <param name="pipelineDescription">Description of the pipeline</param>
         /// <returns>A new element</returns>
         /// <exception cref="GLib.GException">Throws an exception in case of an error</exception>
-        public static Element Launch(string pipelineDescription)
+        public static Element? Launch(string pipelineDescription)
```

Comment:
This does not return null as WrapHandle will throw an exception in case of an error? So this should be `public static Element Launch(string pipelineDescription)`?

Reply:
The actual method gst_parse_launch can return null, so it might be worth switching this to TryWrapHandle.